preload.js

```diff
@@ -129,6 +129,7 @@ window.moment.locale(locale);
 window.Signal = Signal.setup({
   Attachments,
   userDataPath: app.getPath('userData'),
+  getRegionCode: () => window.storage.get('regionCode'),
```

Comment:
Thanks for injecting the function that goes to IndexedDB instead of doing it directly in the message upgrade pipeline 👍

Reply:
Couldn't make the call to IndexedDB from `preload.js` anyway! :0/